Webapp/sources/blueprints/save_reaction/routes.py

```diff
@@ -408,10 +408,14 @@ def clone_reaction() -> Response:
 
     old_reaction = services.reaction.get_current_from_request_form()
 
-    new_reaction_name = request.form['reactionName']
-    workbook_name = request.form["workbook"]
-    workgroup_name = request.form["workgroup"]
-    new_reaction_id = request.form['newReactionID']
+    new_reaction_name = request.form.get('reactionName')
+    workbook_name = request.form.get("workbook")
+    workgroup_name = request.form.get("workgroup")
+    new_reaction_id = request.form.get('newReactionID')
+
+    if None in [new_reaction_name, workbook_name, workgroup_name, new_reaction_id]:
+        return jsonify({'feedback': "ERROR 400 (BAD REQUEST). Please try again."})
```

Comment:
```suggestion
        return jsonify({'feedback': "ERROR 400 (BAD REQUEST). Please try again."}), status.HTTP_BAD_REQUEST
```
using the status stuff mentioned in the comment above